README.md

```diff
@@ -39,7 +61,7 @@ giving us both centralized and local logs.  (Redundancy is nice).
 
 	import (
 		"flag"
-		"github.com/Graylog2/go-gelf/gelf"
+		gelf "gopkg.in/Graylog2/go-gelf/gelf.v1/gelf"
```

Comment:
You should not need the `gelf` at the beginning as the last part of the path is `gelf`.